### PR TITLE
crimson/alienstore: Add douts 

### PR DESCRIPTION
**Pull request by aclamk**
Add ability to tunnel douts from AlienStore(BlueStore) to seastar logger.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by aclamk** [comment]
jenkins test api

File: src/common/ceph_context.cc [hunk 1]
```diff
@@ -679,11 +679,16 @@ int CephContext::_do_command(
 CephContext::CephContext(uint32_t module_type_,
                          enum code_environment_t code_env,
                          int init_flags_)
+  :CephContext(module_type_, create_options{code_env, init_flags_, nullptr})
```
Comment (athanatos) [inline src/common/ceph_context.cc:682]: Space after :

**Review by athanatos — APPROVED** [review]
LGTM except for the formatting nit.  @rzarzynski Can you take a look?

**Comment by athanatos** [comment]
This does need a qa run since it modifies code outside of crimson/

**Comment by tchaikov** [comment]
- [raodos qa run](https://pulpito.ceph.com/kchai-2022-07-27_14:53:26-rados-wip-kefu-testing-2022-07-27-0857-1-distro-default-smithi/)
- [rerun](https://pulpito.ceph.com/kchai-2022-07-27_14:53:26-rados-wip-kefu-testing-2022-07-27-0857-1-distro-default-smithi/) without #41600

failures tracked by https://tracker.ceph.com/issues/56652

File: src/common/ceph_context.cc (hunk 1 above)
Comment (tchaikov) [reply src/common/ceph_context.cc:682]: #47310

**Pull request merged**